twistr/events.go

```diff
@@ -571,23 +634,90 @@ func PlaySummit(s *State, player Aff) {
 	   Region (Europe, Asia, etc.) they Dominate or Control. The player with the
 	   highest modified die roll receives 2 VP and may degrade or improve the
 	   DEFCON level by 1 (do not reroll ties). */
+	regions := []Region{CentralAmerica, SouthAmerica, Europe, MiddleEast, Africa, Asia}
+	playerRoll := SelectRoll(s)
+	oppRoll := SelectRoll(s)
+	for _, region := range regions {
+		sr := ScoreRegion(s, region)
+		if sr.Levels[player] == Domination || sr.Levels[player] == Control {
+			playerRoll++
+		}
+		if sr.Levels[player.Opp()] == Domination || sr.Levels[player.Opp()] == Control {
+			oppRoll++
+		}
+	}
+	if playerRoll == oppRoll {
+		return
+	}
+
+	var winner Aff
+	if playerRoll > oppRoll {
+		winner = player
+	} else {
+		winner = player.Opp()
+	}
+
+	s.GainVP(winner, 2)
+	choice := s.Solicit(winner, "Degrade or improve DEFCON by one level?", []string{"improve", "degrade", "leave it"})
```

Comment:
multi-word choices might be problematic, since the marshal/unmarshaling code breaks on word boundaries.

Best for now to make all choices single-word tokens